data/net/minecraft/core/Direction.mapping

```diff
@@ -9,6 +9,14 @@ CLASS net/minecraft/core/Direction
 		COMMENT Normalized Vector that points in the direction of this Facing
 	FIELD oppositeIndex I
 		COMMENT Index of the opposite Facing in the VALUES array
+	METHOD <init> (Ljava/lang/String;IIIILjava/lang/String;Lnet/minecraft/core/Direction$AxisDirection;Lnet/minecraft/core/Direction$Axis;Lnet/minecraft/core/Vec3i;)V
+		ARG 3 index
+		ARG 4 oppositeIndex
+		ARG 5 horizontalIndex
```

Comment:
So for `ARG 3` and `ARG 5`, they are just being set to field names, `data3d` and `data2d` respectively, so naming them differently could be confusing for users.

I did discuss it in a private channel, but here is a repost of my message:
```java
byte b0, b1;
public thing(byte pitch, byte yaw){
  this.b0 = pitch;
  this.b1 = yar;
}

public byte getB0() {
  return b0;
}
public byte getB1() {
  return b1;
}
```

Sure the values may actually be `pitch` and `yaw`, but if someone was using it, their IDE would show them this, and be pretty confusing as to what is what.
![image](https://user-images.githubusercontent.com/5553605/124281373-f128a380-db49-11eb-9d6a-45039b2fb4e4.png)

I think doing `@param data3d <explanation here>` would be the best way to go about explaining what it does.